.travis.yml

```diff
@@ -12,13 +12,21 @@ matrix:
       sudo: required
       dist: trusty
     - os: linux
-      env: SWIGLANG= SWIG_CC=gcc-5 SWIG_CXX=g++-5 CPP11=1
+      env: SWIGLANG= SWIG_CC=gcc-5 SWIG_CXX=g++-5 SWIG_FC=gfortran-5 CPP11=1
```

Comment:
This test and the next one is for purely building SWIG without any language tests (SWIGLANG is empty) and testing default configure. Remove SWIG_FC.